Libraries/LibThread/Thread.h

```diff
@@ -41,12 +42,14 @@ class Thread final : public Core::Object {
     virtual ~Thread();
 
     void start();
-    void quit(void *code = 0);
+    void quit(void* code = 0);
+    void join();
 
 private:
     Function<int()> m_action;
     pthread_t m_tid;
     String m_thread_name;
+    AK::Atomic<bool> m_is_running;
```

Comment:
```suggestion
    Atomic<bool> m_is_running;
```

One of the SerenityOS idiosyncrasies is that in AK, things are declared inside the `namespace AK`, but usually imported back into the public namespace, and then the rest of the code basically never uses the `AK::` prefix.